### DISCUSSION TITLE
Where can i find latest rustdesk setup file for 32 bit windows OS???

### DISCUSSION THREAD

**Discussion by riddler11111** [rustdesk/rustdesk#4986] [General]
Where can i find latest rustdesk setup file for 32 bit windows OS???

Unable to find RustDesk 1.2.0 or 1.2.1 setup file for 32 bit OS.

**Comment by dinger1986** [comment]
https://github.com/rustdesk/rustdesk/releases/download/1.2.1/rustdesk-1.2.1-x86-sciter.exe

**Comment by Chapra78** [comment]
Hello guys,

We have deployed a self hosted Ruskdesk server (192.168.1.10) in our LAN.
Then we have build the Rustdesk Client using the sourcecode downloaded from "https://github.com/rustdesk/rustdesk" and by following the steps mentioned at https://github.com/infiniteremote/installer/files/14814894/Setup_rustdesk_development_on_Windows.pdf
We have hardcoded our server IP (192.168.1.10) and Rustdesk server key in the source code.
The client is working successfully on all our 64 bit windows machines.Unfortunately we are having issues running the Rustdesk client on 32 bit windows machines. Hence please let us know how to resolve this issue and how to build the client for running on 32 bit Windows machines.
